Inputs.md

```diff
@@ -0,0 +1,94 @@
+This guide will describe how inputs from the user (keyboard, mouse, etc.) are routed through InputWritr and how the DeviceLayr Gamepad can be used on top of it.
+
+### Table of Contents
+1. [InputWritr](#inputwritr)
+    1. [Events and Triggers](#events-and-triggers)
+    2. [History](#history)
+2. [DeviceLayr](#devicelayr)
+
+# InputWritr
+
+## Events and Triggers
+
+InputWritr is a module that automates interactions with user inputs and events. 
+
+Events are the game's responses to user input.
+They allow the user to take control of some part of the game or have interaction with it.
+Events can be added using `addEvent` which takes in a trigger name, key code, and the function to run the event is triggered; they are removed with `removeEvent`
```

Comment:
Instead of this long run-on sentence that explains two concepts, followed by a code sample that shows two concepts, have one sentence with one code sample for each concept.